app/services/hyrax/valkyrie_upload.rb

```diff
@@ -28,7 +28,7 @@ def self.file(
     )
     io.close
 
-    file_metadata = find_or_create_metadata(id: streamfile.id, file: streamfile)
+    file_metadata = Hyrax::FileMetadata(file: streamfile)
```

Comment:
```suggestion
    file_metadata = Hyrax::FileMetadata(streamfile)
```
That should resolve the `NoMethodError:
  undefined method 'id' for #<Hash>` errors.